### DISCUSSION TITLE
replacement updates:  pr body columns and merge confidence badges

### DISCUSSION THREAD

**Discussion by setchy** [renovatebot/renovate#21381] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.36.0

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Cloud

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

Wanted to start a discussion about how the PR table behaves with relation to replacement updates.

There are items that stand out

1. the table does not mention the new package name
2. the merge confidence badge urls contain the previous package name details

![Screenshot 2023-04-07 at 7 45 35 AM](https://user-images.githubusercontent.com/386277/230603996-1551b0e4-b9dc-4284-be3d-d7a5a302e404.png)

![Screenshot 2023-04-07 at 7 52 20 AM](https://user-images.githubusercontent.com/386277/230604418-8ddd0f39-2466-4433-9cb0-9f004a545bf4.png)

### What you tried so far.

-

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
Merge confidence should avoided for replacements

**Reply by setchy** [reply]
Ok. I've updated a rule to exclude those columns for replacement updates.

What about the package name before -> after. Would be nice if that behaved like the Changes column
